.spin/cmds.py

```diff
@@ -263,6 +263,38 @@ def _run_asv(cmd):
 
     util.run(cmd, cwd='benchmarks', env=env)
 
+@click.command()
+@click.option(
+    "-b", "--branch",
+    metavar='branch',
+    default="main",
+)
+@click.option(
+    '--uncommitted',
+    is_flag=True,
+    default=False,
+    required=False,
+)
+@click.pass_context
+def lint(ctx, branch, uncommitted):
+    """🔦 Run lint checks on diff.
+    Provide target branch name or `uncommitted` to check before committing:
+
+    \b
+    Examples:
+
+    \b
+    $ spin lint
+    $ spin lint --branch main
+    $ spin lint uncommitted
```

Comment:
missing `--` in `--uncommitted`

Reply:
Maybe add a short paragraph above these examples explaining what the flags do?

Reply:
Yep, will do.